samples/Nancy.Demo.Hosting.Kestrel/HomeModule.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Nancy.Demo.Hosting.Kestrel
+{
+    using System.Threading.Tasks;
+
+    public class HomeModule : NancyModule
+    {
+        public HomeModule()
+        {
+            Get["/"] = (o, token) => Task.FromResult<dynamic>("Hello from Nancy running on CoreCLR");
+
+            Get["/conneg/{name}"] = (parameters, token) => Task.FromResult<dynamic>(new Person() { Name = parameters.name });
+        }
+    }
+
+    public class Person

```

Comment:
just because it's a sample, doesn't mean you should put multiple classes in the same file. tut tut tut.